contracts/libraries/TokenLib.sol

```diff
@@ -12,6 +12,24 @@ library TokenLib {
 
     using SafeMath for uint256;
 
+    struct EIP712Domain {
+        string  name;
```

Comment:
Do we need to add the `version` here to avoid to use the same messages when the security token is upgradable?

Reply:
If the ST is upgraded, the address (verifying address) will change and hence this sig will become invalid.
Also, these signs can only be used by the owners themselves so there is no question of reuse.

Reply:
`verifyingContract` will not change if we upgrade the ST using the proxy. or it will refer to logic contract address.

Reply:
1) We can not upgrade ST using proxy.
2) If we upgrade using proxy, these signatures will still remain useless because these signatures are used to trigger an irrevocable action. If the action has already been triggered, it can't be triggered again anyway. The information regarding triggering is stored in the proxy so a triggered action will remain triggered.